dvc/command/experiments.py

```diff
@@ -552,3 +613,156 @@ def add_parser(subparsers, parent_parser):
         metavar="<n>",
     )
     experiments_diff_parser.set_defaults(func=CmdExperimentsDiff)
+
+    EXPERIMENTS_RUN_HELP = (
+        "Reproduce complete or partial experiment pipelines."
+    )
+    experiments_run_parser = experiments_subparsers.add_parser(
```

Comment:
all experiments run/repro behavior will be moved from `dvc repro` to `dvc exp run` in a follow-up PR, for now `repro -e` is just duplicated here so that checkpoint runs work correctly